mne/time_frequency/tests/test_tfr.py

```diff
@@ -99,6 +101,27 @@ def test_time_frequency():
     assert_true(np.sum(itc.data >= 1) == 0)
     assert_true(np.sum(itc.data <= 0) == 0)
 
+    # grand average
+    itc2 = itc.copy()
+    itc2.info['bads'] = [itc2.ch_names[0]]  # test channel drop
+    gave = grand_average([itc2, itc2.copy()], drop_bads=True)
```

Comment:
I'm confused. With this setting both instances are equal. If you would use the original itc, you would have sets with different 'bads', which would be a better test in my opinion. Also that would avoid the unnecessary copying.